### DISCUSSION TITLE
login `next` query param does not match the base_url

### DISCUSSION THREAD

**Discussion by mattiagiupponi** [apache/airflow#39720] [Q&A]
I deployed my airflow application behind a `Microsoft application gateway`. I updated the `base_url` to match the public IP provided by the application gateway, for example, `1.2.3.4`.

However, when I try to access the login page, the "next" query parameter refers to the internal IP of the machine (for example, `192.0.0.0`) instead of the public IP of the Application Gateway.

So for instance, I have this value as base_url

```
BASE_URL=http://1.2.3.4/airflow
```

When see the login page the URL looks like this:
```
http://1.2.3.4/airflow/login/?next=http%3A%2F%2F192.0.0.0%2Fairflow%2Fhome
```
I was expecting to see something like this:
```
http://1.2.3.4/airflow/login/?next=http%3A%2F%2F1.2.3.4%2Fairflow%2Fhome
```
Is there some additional configuration that I have to set?

NOTE: i'm not using kubernetes, but plain virtual machines
aiflow 2.6.2

**Comment by mattiagiupponi** [comment]
bump

**Comment by Taragolis** [comment]
> aiflow 2.6.2

I guess you need to try whether it fixed in never version of Airflow or not
